packages/performance-tests/test/TestTransformerNodule.ts

```diff
@@ -0,0 +1,11 @@
+import { IModelDb } from "@itwin/core-backend";
+import { IModelTransformer } from "@itwin/imodel-transformer";
+
+/**
+ * the type of a module imported by the tests holding a custom transformer
+ * implementation to test
+ */
+export interface TestTransformerModule {
+  createIdentityTransform?(sourceDb: IModelDb, targetDb: IModelDb): Promise<IModelTransformer>;
```

Comment:
didn't we want to change these from returning an IModelTransformer to a generic `{ run: () => Promise<void> }` object?